### DISCUSSION TITLE
How can I use Link with custom action?

### DISCUSSION THREAD

**Discussion by Kation** [Megabit/Blazorise#4931] [Q&A] [answered]
I want to set `To` property so that I can copy link in browser.
But I want to handle click in custom code and pervent browser navigate to new url.

**Comment by stsrki** [comment, marked as answer]
Unfortunately, it is impossible to prevent `Link` component click because we're already doing it behind-the-scenes.

In your case it is best to use a regular `<a href>` element with `@onclick:preventDefault=true`.
